src/javascript.rs

```diff
@@ -119,6 +126,41 @@ impl<'a> Generator<'a> {
             "}",
         ])
     }
+
+    fn external_function<T>(
+        &mut self,
+        public: bool,
+        name: &'a str,
+        arguments: &'a [ExternalFnArg<T>],
+        module: &'a str,
+        fun: &'a str,
+    ) -> Output<'a> {
+        let head = if public {
+            "export function "
+        } else {
+            "function "
+        };
+        let arguments = wrap_args(arguments.iter().enumerate().map(|a| match a {
+            (index, ExternalFnArg { label, .. }) => {
+                let arg_name = label.clone().unwrap_or(format!("arg{}", index));
+                Document::String(arg_name)
+                // label
```

Comment:
This is the code, from you suggestion, moved after merging main. I wasn't sure how to use it so have left my original code as it was functioning.

Reply:
This would be the new definition of `arg_name`